engine/common/api_helpers/mixins.py

```diff
@@ -351,9 +354,19 @@ def get_attr_template(self, attr, alert_receive_channel, render_for=None):
                 return Response({"preview": e.fallback_message}, status.HTTP_200_OK)
         else:
             templated_attr = None
-        response = {"preview": templated_attr}
+        response = {"preview": templated_attr, "is_valid_json_dict": self.is_valid_json_dict(templated_attr)}
```

Comment:
Could you please update the [response schema](https://github.com/grafana/oncall/pull/3643/files#diff-ed4d02ac8c4dc7e83cc8c73c852e084b30a82f0425d1149f39768d41ac44f9fcR300)?

Reply:
Updated